cluster/ci/config/prow/plugins.yaml

```diff
@@ -328,6 +328,18 @@ plugins:
   operator-framework/operator-marketplace:
   - approve
 
+  operator-framework/operator-metering:
+  - approve
+
+  operator-framework/presto:
+  - approve
+
+  operator-framework/hadoop:
+  - approve
+
+  operator-framework/hive:
+  - approve
+
```

Comment:
This configures plugins for four different repos but the PR only adds jobs for one of them. Is that expected? If you plan to add jobs for the remaining repos later, it might be better to put all repo-specific config into a single PR, it makes reviews easier.

Also, for "full" CI config, you also need to configure automated merges by including your repos in the `cluster/ci/config/prow/config.yaml` file, in the appropriate query under `tide` configuration.

Reply:
I'm planning on adding more jobs and have other commits unpushed because I'm trying to minimize change to get 1 working first. I can split that up though.